### ISSUE TITLE
Bug [1.20.1 bettermc mod pack]: server/game crash

### ISSUE THREAD

**Issue by drsidegg** [Zepalesque/Aether-Redux-Issue-Tracker#32]
**What version of the mod, Forge/Neoforge and Minecraft are you using?**
mc:1.20.1
redux: 1.3

**Describe the bug**
Playing the bettermc modpack, my server crashes with ... caused by 
...net.zepalesque.aether.world.feature.FieldsproutTreeFeature... 
whenever I try to start it

**To Reproduce**
Going in the aether biome and maybe expect a crash?
I opened a chest and the game crashed instantly

**Expected behavior**
I was looting a "bronze dungeon" and when I opened one of the fake chest the game instantly crashed, nothing in the crash txt file mentions anything

**Screenshots**

**Crash report (if necessary and/or applicable)**
ex: [upload
[crash-2023-11-08_23.25.59-server.txt](https://github.com/Zepalesque/Aether-Redux-Issue-Tracker/files/13303798/crash-2023-11-08_23.25.59-server.txt)
 file here]

**Log file (should be titled latest.log, if applicable)**

**Additional context**
This bug seems to be coming from a new version since someone already made a bug report 3 days ago...?
Sorry I'm quite in a hurry because I'm writing this at midnight and I'm tired but I'll give more details when I wake up

**Comment by drsidegg** [comment]
Fixed!

Try to get a friend to join the server and give them op via cmd if you're hosting the server by yourself

2: tell them to do the command /save_all 
shutdown the server and install the following program
https://github.com/jaquadro/NBTExplorer/releases
this will let you access your playerfile and edit your location in game
![image](https://github.com/Zepalesque/Aether-Redux-Issue-Tracker/assets/146892220/cd949d1a-6b88-4e05-800f-83fe172c01de)

changethe post to 0 0 0 and then it should work!

**Comment by TigerWalts** [comment]
```
java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
```

The `Apache Commons Compress` library is not installed and loaded on dedicated Forge servers. You'll need to use something else like `java.util.ArrayList`

**Comment by drsidegg** [comment]
> ```
> java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
> ```
> 
> The `Apache Commons Compress` library is not installed and loaded on dedicated Forge servers. You'll need to use something else like `java.util.ArrayList`

And how do I exactly do that?

**Comment by Zepalesque** [comment]
actually rereading and looking at the crash log it seems like the chest thing was likely just coincidencal timing and it’s still the fieldsprout bug, so yeah that was fixed in v1.3.1

**Comment by drsidegg** [comment]
> actually rereading and looking at the crash log it seems like the chest thing was likely just coincidencal timing and it’s still the fieldsprout bug, so yeah that was fixed in v1.3.1

Thank you!!! Yea I’ll just update the mod to the newest version by myself since I am hosting the server, also the chest probably « banned » the chunk I was in. Can a sprout spawn in dungeon chest by any chance?
